src/main/java/GeePee.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+public class GeePee {
+
+    //constant to determine required padding to extract the name of a Todo object
+    public static final int TODO_PADDING = 5;
+
+    //constants to determine required padding to extract the name of a Deadline object,
+    //and when it needs to be completed by
+    public static final int DEADLINE_PADDING = 9;
+    public static final int BY_PADDING = 4;
```

Comment:
I like the use of constants as it improves readability and makes it easier to manage changes